src/autocomplete.js

```diff
@@ -611,20 +619,33 @@ class CompletionProvider {
             // TODO add support for options.deleteSuffix
```

Comment:
Could you make the previous if case return (including `editor.endOperation();`)?

Reply:
Sorry, I don't understand your request. Do you mean 
```javascript
if (!this.completions) {
   editor.endOperation();
   return false;
}
```
to end already started operation in case there is no completions?